subscribing-java-testing/src/test/java/com/ably/tracking/example/javasubscriber/UsageExamples.java

```diff
@@ -78,4 +83,18 @@ public void subscriberFacadeUsageExample() {
             // handle interruption exception
         }
     }
+
+    private Authentication createTokenRequestAuthentication() {
+        return AuthenticationFacade.tokenRequest(tokenParams -> CompletableFuture.supplyAsync(() -> {
+            // get the token from you auth servers
+            return new EmptyTokenRequest();
+        }));
+    }
+
+    private Authentication createJwtAuthentication() {
+        return AuthenticationFacade.jwt(tokenParams -> CompletableFuture.supplyAsync(() -> {
+            // get the token from you auth servers
+            return "created-jwt";
```

Comment:
This could perhaps be clearer to indicate that it's a placeholder - even the dashes, at the moment, make it look more 'official'. Perhaps:

```suggestion
            return "DUMMY PLACEHOLDER: RETURN JWT FROM AUTH SERVER HERE";
```

Reply:
https://github.com/ably/ably-asset-tracking-android/pull/787/commits/ccf94c19bd2fbb41201098788c608b5ba2411b57